shared/common-adapters/text.meta.desktop.js

```diff
@@ -259,6 +264,38 @@ const metaData: {[key: TextType]: MetaType} = {
       wordWrap: 'break-word',
     },
   },
+  'CodeSnippet': {
```

Comment:
not clear to me we should put component specific styles in here. Maybe better to just use it in the component itself